dapps/onboarding.vue/src/i18n/de.json

```diff
@@ -14,6 +14,10 @@
         "1": {
           "desc": "Die Absicherung Ihrer Daten ist uns sehr wichtig. Daher ist der Login-Prozess zweigeteilt. Bitte geben Sie Ihr Passwort ein, um Ihr Profil zu entsperren und mit Ihrer Arbeit fortzufahren.<br><br><b>Bitte beachten:</b> Aus Sicherheitsgründen, müssen Sie ihr Passwort nach jedem neu laden der Seite erneut eingeben. Somit bleiben Sie vollständig \"Self-Sovereign\" und Herr Ihrer Daten.",
           "title": "Willkommen im evan.network"
+        },
+        "2": {
+          "desc": "Die Absicherung Ihrer Daten ist uns sehr wichtig. Daher ist der Login-Prozess zweigeteilt. Bitte geben Sie Ihr Passwort ein, um Ihr Profil zu entsperren und mit Ihrer Arbeit fortzufahren.<br><br><b>Bitte beachten:</b> Aus Sicherheitsgründen, müssen Sie ihr Passwort nach jedem neu laden der Seite erneut eingeben. Somit bleiben Sie vollständig \"Self-Sovereign\" und Herr Ihrer Daten.",
```

Comment:
This is exactly the same as above isn't there a cleaner solution for this?

Reply:
Thats true. Currently we don't have any texts for this case. I will introduce a new one